### PR TITLE
SAK-45712: Tests & Quizzes: Display trash icon instead of underlined …

### PR DESCRIPTION
**Pull request by hornersa**
…'X' for deleting student submissions

### CODE REVIEW

File: samigo/samigo-app/src/webapp/jsf/evaluation/totalScores.jsp [hunk 1]
```diff
@@ -276,7 +276,7 @@ $(document).ready(function(){
        <h:outputText value="\"></a>" escape="false" />
 
        <h:commandLink styleClass="sam-scoretable-deleteattempt" title="#{commonMessages.delete_attempt}" action="totalScores" immediate="true" rendered="true" >
-         <h:outputText value="X" rendered="#{description.submittedDate!=null &&  description.assessmentGradingId ne '-1'}" />
+         <h:outputText value="" styleClass="fa fa-trash" rendered="#{description.submittedDate!=null &&  description.assessmentGradingId ne '-1'}" />
```
Comment (ottenhoff) [inline samigo/samigo-app/src/webapp/jsf/evaluation/totalScores.jsp:279]: Thanks for the PR.  As we're moving from text to an icon, we need to provide an alternative for screen readers. The easiest way is probably via an sr-only div with "Delete submission" as the text

Comment (hornersa) [reply samigo/samigo-app/src/webapp/jsf/evaluation/totalScores.jsp:279]: Good catch. I'll see if I can borrow (some of) the markup used for implementing the "Copy to clipboard" button now present in the published Settings view.

File: samigo/samigo-app/src/webapp/jsf/evaluation/totalScores.jsp [hunk 2]
```diff
@@ -276,7 +276,8 @@ $(document).ready(function(){
        <h:outputText value="\"></a>" escape="false" />
 
        <h:commandLink styleClass="sam-scoretable-deleteattempt" title="#{commonMessages.delete_attempt}" action="totalScores" immediate="true" rendered="true" >
-         <h:outputText value="X" rendered="#{description.submittedDate!=null &&  description.assessmentGradingId ne '-1'}" />
+           <h:panelGroup rendered="#{description.submittedDate!=null &&  description.assessmentGradingId ne '-1'}"><span class="fa fa-trash" aria-hidden="true"></span></h:panelGroup>
+	   <span class="sr-only"><h:outputText value="#{commonMessages.delete}" rendered="#{description.submittedDate!=null &&  description.assessmentGradingId ne '-1'}" /></span>
```
Comment (ottenhoff) [inline samigo/samigo-app/src/webapp/jsf/evaluation/totalScores.jsp:280]: would be simpler to just move the span and outputText inside the panelGroup on line above... then you just have one if statement

**Review by ottenhoff — APPROVED** [review]
awesome!

**Review by mpellicer — APPROVED** [review]

**Pull request merged**